src/backend/marsha/core/tests/test_api_video_live_participants.py

```diff
@@ -0,0 +1,379 @@
+                "medialive": {
+                    "input": {
+                        "id": "medialive_input_1",
+                        "endpoints": [
+                            "https://live_endpoint1",
+                            "https://live_endpoint2",
+                        ],
+                    },
+                    "channel": {"id": "medialive_channel_1"},
+                },
+                "mediapackage": {
+                    "id": "mediapackage_channel_1",
+                    "endpoints": {
+                        "hls": {
+                            "id": "endpoint1",
+                            "url": "https://channel_endpoint1/live.m3u8",
+                        },
+                    },
+                },
+            },
+        )
+
+        jwt_token = AccessToken()
+        jwt_token.payload["resource_id"] = str(video.id)
+        jwt_token.payload["roles"] = [random.choice(["instructor", "administrator"])]
+        jwt_token.payload["permissions"] = {"can_update": True}
+        jwt_token.payload["user"] = {"id": "56255f3807599c377bf0e5bf072359fd"}
+
+        data = {
+            "participants_asking_to_join": [
+                {
+                    "id": "1",
+                },
+            ],
+            "participants_in_discussion": [
+                {
+                    "name": "Instructor",
+                },
+            ],
+        }
+
+        with mock.patch.object(
+            channel_layers_utils, "dispatch_video_to_groups"
+        ) as mock_dispatch_video_to_groups, mock.patch(
+            "marsha.core.serializers.xmpp_utils.generate_jwt"
+        ) as mock_jwt_encode:
+            mock_jwt_encode.return_value = "xmpp_jwt"
+            response = self.client.patch(
+                f"/api/videos/{video.id}/participants/",
+                HTTP_AUTHORIZATION=f"Bearer {jwt_token}",
+                data=data,
+                content_type="application/json",
+            )
+            video.refresh_from_db()
+            mock_dispatch_video_to_groups.assert_not_called()
+
+            self.assertEqual(response.status_code, 400)
+
+            content = response.json()
+            self.assertEqual(
+                content,
+                {
+                    "participants_asking_to_join": [
+                        {"name": ["This field is required."]}
+                    ],
+                    "participants_in_discussion": [{"id": ["This field is required."]}],
+                },
+            )
+
+    @override_settings(LIVE_CHAT_ENABLED=True)
+    @override_settings(XMPP_BOSH_URL="https://xmpp-server.com/http-bind")
+    @override_settings(XMPP_CONFERENCE_DOMAIN="conference.xmpp-server.com")
+    @override_settings(XMPP_DOMAIN="conference.xmpp-server.com")
+    @override_settings(XMPP_JWT_SHARED_SECRET="xmpp_shared_secret")
+    def test_api_video_participants_instructor_invalid_empty(self):
+        """An invalid request must return a 400 error."""
```

Comment:
Can you add 2 other scenarios ? One with other data for a participant (`id` + `name` + an extra data). And an other one with empty participants ?
```
        data = {
            "participants_asking_to_join": [],
            "participants_in_discussion": [],
        }
```